spine-ts/spine-phaser/src/SpinePlugin.ts

```diff
@@ -257,6 +257,16 @@ enum SpineSkeletonDataFileType {
 
 class SpineSkeletonDataFile extends Phaser.Loader.MultiFile {
 	constructor (loader: Phaser.Loader.LoaderPlugin, key: string, url: string, public fileType: SpineSkeletonDataFileType, xhrSettings: Phaser.Types.Loader.XHRSettingsObject) {
+		
+		if (Phaser.Utils.Objects.IsPlainObject(key as any)) {
```

Comment:
This cast validates the `key` parameters type, which is `string`. Can you provide an interface definition for the object?

Reply:
Ok, I did it.